jupyterlab/extensions/pypi.py

```diff
@@ -49,24 +50,32 @@ def make_connection(self, host):
         return connection
 
 
-xmlrpc_transport_override = None
-
 all_proxy_url = environ.get("ALL_PROXY")
 # For historical reasons, we also support the lowercase environment variables.
 # Info: https://about.gitlab.com/blog/2021/01/27/we-need-to-talk-no-proxy/
 http_proxy_url = environ.get("http_proxy") or environ.get("HTTP_PROXY") or all_proxy_url
 https_proxy_url = (
     environ.get("https_proxy") or environ.get("HTTPS_PROXY") or http_proxy_url or all_proxy_url
 )
+
+# sniff ``httpx`` version for version-sensitive API
+_httpx_version = Version(httpx.__version__)
+_httpx_client_args = {}
+
 proxies = None
+xmlrpc_transport_override = None
 
 if http_proxy_url:
     http_proxy = urlparse(http_proxy_url)
     proxy_host, _, proxy_port = http_proxy.netloc.partition(":")
 
     proxies = {
         "http://": httpx.HTTPTransport(proxy=http_proxy_url),
-        "https://": httpx.HTTPTransport(proxy=https_proxy_url),
+        "https://": httpx.HTTPTransport(proxy=http_proxy_url),
```

Comment:
I do not follow the reason for removal of `s` here:

```suggestion
        "https://": httpx.HTTPTransport(proxy=https_proxy_url),
```